database/tables/users.pg

```diff
@@ -0,0 +1,32 @@
+columns
+    name: text
+    uid: text not null
+    uin: character(9)
+    user_id: bigint not null default nextval('users_user_id_seq'::regclass)
+
+indexes
+    users_pkey: PRIMARY KEY (user_id)
+    users_uid_key: UNIQUE (uid)
+    users_uin_key: UNIQUE (uin)
```

Comment:
This line (`users_uin_key`) currently doesn't match between a migrated old DB and a fresh DB. I know how to fix it, but I have run out of time now. I can do it later today.